build.sh

```diff
@@ -21,10 +21,11 @@
 
 set -eE
 
-meson_packages="ssp winpthreads zlib libpng libjpeg_turbo libtiff libopenjp2 sqlite3 proxy_libintl libffi pcre2 glib gdk_pixbuf pixman cairo libxml2"
+meson_packages="libdicom ssp winpthreads zlib libpng libjpeg_turbo libtiff libopenjp2 sqlite3 proxy_libintl libffi pcre2 glib gdk_pixbuf pixman cairo libxml2"
```

Comment:
libdicom isn't a dependency of any of the other libraries, so I think it'd be clearer to put it at the end of the list.  Also we'll need to track `uthash` in this file, and `check` if we continue needing it.

Reply:
Sorry, I should have said: after the other dependencies but before OpenSlide.

Reply:
OK, done!